index.html

```diff
@@ -140,8 +140,15 @@
     <section id="terminology">
     <h2>Terminology</h2>
     <p>
-      Terminology is based on W3C's Linked Data Platform 1.0 [[LDP]] and Memento [[RFC7089]].
+      Terminology is based on W3C's Linked Data Platform 1.0 [[LDP]], Memento [[RFC7089]], and Web Access Control
+      [[SOLIDWAC]].
```

Comment:
My understanding is that the terminology section is normative, so I think the external references should be `[[!LDP]]`, `[[!RFC7089]]`, and `[[!SOLIDWEBAC]]`.

Also, it's `[[!SOLIDWEBAC]]` not `[[!SOLIDWAC]]`